securetar/__init__.py

```diff
@@ -385,7 +385,7 @@ def _is_excluded_by_filter(path: PurePath, exclude_list: list[str]) -> bool:
     """Filter to filter excludes."""
 
     for exclude in exclude_list:
-        if not path.match(exclude):
+        if not path.full_match(exclude):
```

Comment:
That not work as we work with PurePath (without any blocking stats). That is the nice thing on that function to keep it fast

Reply:
the `full_match` method with improved glob support was added in Python 3.13: https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.full_match